subgraphs/seaport/src/helper.ts

```diff
@@ -1,4 +1,32 @@
-import { Address, BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+import { Address, BigDecimal, BigInt, Bytes, ethereum, log } from "@graphprotocol/graph-ts";
+
+export namespace MethodSignatures {
+  export const FULFILL_BASIC_ORDER = "0XFB0F3EE1"
+  export const FULFILL_ORDER = "0XB3A34C4C"
+  export const FULFILL_ADVANCED_ORDER = "0XE7ACAB24"
+  export const FULFILL_AVAILABLE_ORDERS = "0XED98A574"
+  export const FULFILL_AVAILABLE_ADVANCED_ORDERS = "0X87201B41"
+  export const MATCH_ORDERS = "0XA8174404"
+  export const MATCH_ADVANCED_ORDERS = "0X55944A42"
+}
+
+export namespace OrderFulfillmentMethods {
+  export const FULFILL_BASIC_ORDER = "FULFILL_BASIC_ORDER" 
+  export const FULFILL_ORDER = "FULFILL_ORDER" 
+  export const FULFILL_ADVANCED_ORDER = "FULFILL_ADVANCED_ORDER" 
+  export const FULFILL_AVAILABLE_ORDERS = "FULFILL_AVAILABLE_ORDERS"
+  export const FULFILL_AVAILABLE_ADVANCED_ORDERS = "FULFILL_AVAILABLE_ADVANCED_ORDERS"
+  export const MATCH_ORDERS = "MATCH_ORDERS" 
+  export const MATCH_ADVANCED_ORDERS = "MATCH_ADVANCED_ORDERS" 
+}
+
+export namespace MethodSignatureTypeStrings {
```

Comment:
is this being used anywhere?

Reply:
@this-username-is-taken Sure, I'll get these fixed over the next week.